packages/components/src/ConnectionCard/HorizontalDash.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import * as React from 'react';
+import { View, Dimensions } from 'react-native';
+import { StyleSheet } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+type Props = {|
+  +size?: number,
+  +borderColor?: string,
+  +overlayColor?: string,
+|};
+
+export default function HorizontalDash({
+  size = 2,
+  borderColor = defaultTokens.borderColorCard,
+  overlayColor = defaultTokens.backgroundCard,
+}: Props) {
+  const dashStyles = { borderWidth: size, borderColor: borderColor };
+  const overlayStyles = {
+    bottom: -size,
+    left: -size,
+    height: size,
+    backgroundColor: overlayColor,
+  };
+  return (
+    <View style={[styles.dash, dashStyles]}>
+      <View style={[styles.dashOverlay, overlayStyles]} />
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  dash: {
+    width: '100%',
+    height: 0,
+    borderStyle: 'dotted',
```

Comment:
```suggestion
    borderStyle: 'dotted',
        web: {
      borderLeftStyle: 'none',
      borderRightStyle: 'none',
      borderBottomStyle: 'none',
    },
```

Otherwise, it looks really big on web version:
![screenshot 2019-01-10 at 11 56 38](https://user-images.githubusercontent.com/22741774/50964138-cf955680-14ce-11e9-88ca-0e60d88f2fdc.png)